sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -254,3 +254,13 @@ CREATE TABLE `form_therapy_groups_attendance` (
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB ;
 #EndIf
+
```

Comment:
Don't add it like this. Instead modify the related upgrade code block above (and do not hard-code pc_catid; just allow the insert to create it since it is an autoincrement field).